internal/gcsx/prefix_bucket_test.go

```diff
@@ -364,6 +364,67 @@ func (t *PrefixBucketTest) TestCreateObjectChunkWriterAndFlushPendingWrites() {
 	assert.Equal(t.T(), string(content), string(actual))
 }
 
+func (t *PrefixBucketTest) TestCreateAppendableObjectWriterAndFlush() {
+	var err error
+	suffix := "taco"
+	content := []byte("foobar")
+
+	// Create the object writer.
+	w, err := t.bucket.CreateAppendableObjectWriter(
+		t.ctx,
+		&gcs.CreateObjectChunkWriterRequest{
+			CreateObjectRequest: gcs.CreateObjectRequest{
+				Name: suffix,
+			},
+			ChunkSize: 1024,
+			Offset:    10,
+		})
+
```

Comment:
Nit: remove hte new line here, this is part of test invocation only.

Same for below method